### PR TITLE
Iteration3 save

### PR DESCRIPTION
**Pull request by kimalajoy**
Updated save button functionality so it saves the name of the outfit and creates a new card in our saved outfits section and updated the class instance so hat items get pushed() into the garments array. (We will need to figure out how to get the other garments to push into this same array and then figure out how to only get one item from each section to be in the array at once!)

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -1,13 +1,21 @@
-var outfit = new Outfit({id: generateRandomId()});
+// Outfit class instance
+var outfit = new Outfit(generateRandomId());
 
 function generateRandomId() {
   return Math.floor(Math.random() * 1000);
 }
-//button lists
+console.log(outfit)
```
Comment (collinkallery) [inline main.js:7]: Let's go ahead and delete this! :)

File: main.js [hunk 2]
```diff
@@ -17,13 +25,62 @@ var hatImages = document.querySelectorAll('.hat');
...
 backgroundButtons.addEventListener('click', backgroundButtonEvents)
 
 
-function hatButtonEvents() {
+// This function is creating a new child <div> element for the saved outfit
+// the img src and alt are the children of the div
+// add the newSavedOutfitCard (child) to the outfit-section div (parent) element
+// textnode is a child as well - it's the text that is entered into the save input
+function createOutfitCard(saveOutfitName){
+  const newSavedOutfitCard = document.createElement('div');
```
Comment (collinkallery) [inline main.js:53]: Nice use of "const" here!!!

File: main.js [hunk 3]
```diff
@@ -58,6 +115,7 @@ function addHat(event) {
 };
 
 function highlightClothesButtons(event) {
+  console.log(event);
```
Comment (collinkallery) [inline main.js:118]: Let's delete this console log too.

File: main.js (hunk 1 above)
Comment (kimalajoy) [reply main.js:7]: Whoops! I know I missed a couple of these!

**Pull request merged**